src/fsharp/symbols/Symbols.fs

```diff
@@ -290,6 +290,9 @@ type FSharpSymbol(cenv: SymbolEnv, item: (unit -> Item), access: (FSharpSymbol -
         | Item.ArgName(id, ty, _)  ->
              FSharpParameter(cenv, ty, {Attribs=[]; Name=Some id}, Some id.idRange, isParamArrayArg=false, isInArg=false, isOutArg=false, isOptionalArg=false) :> _
 
+        | Item.ImplicitOp(_, { contents = Some(TraitConstraintSln.FSMethSln(_, vref, _)) }) ->
```

Comment:
This is a good fix, though we should also cover the case where the solution is a .NET Method or an F# record field (the other possibliies for TestConstraintSln)

@7sharp9 Would you be able to submit a new PR for those cases?

Reply:
@dsyme , yeah I can add the others too.
Testing these is really tough though as Ive only been able to replicate the issue within Falanx referencing a Fleece operator, which references FSharpPlus operator.  i.e.No simple minimal repo.  There may be other symbols that we could surface too...

Reply:
Incidentally @dsyme Do you know why `ItemWithInst.Typar` ia always empty for custom operators?

https://github.com/Microsoft/visualfsharp/blob/a26d32a86c88f4ae61a77bfe5c764cb0ea8cec8b/src/fsharp/NameResolution.fsi#L133

Reply:
Custom operators (in computation expressions) or implicit operators?

Reply:
Implicit operators, they seem to match always get created with `ItemWithNoInst`